### ISSUE TITLE
A config file for being able to locally deploy the bot

### ISSUE THREAD

**Issue by HesamKorki** [anvaari/BarnameKon#4]
A config file for Heruko credentials and other configs needs to be created and handled in the code in a way that deploying on a local machine for testing purposes is available.

**Comment by anvaari** [comment]
You are right. Actually I totally forgot that. I will add those file as soon as possible. Thanks :)

**Comment by anvaari** [comment]
Honestly, I haven't much experience in software development. All I know is I should provide .env file in order to locally deploy app. But when I go forward according to [this](https://devcenter.heroku.com/articles/heroku-local) tutorial, Bot don't answer to my request. Do you have any idea ?
@HesamKorki

**Comment by anvaari** [comment]
Finally, I Find out. Please follow these steps in order to deploy locally.
- Install Heroku cli [From Here](https://devcenter.heroku.com/articles/heroku-cli#download-and-install). You should change your IP for sign up!
- Download [Ngrok](https://ngrok.com/download) and set it on port 5000 (`./ngrok http 5000`)
- Clone Repository 
- Change variable in .env.sample in proper way
- Rename .env.sample to .env
- In Barnamekon.py : 
   - See line 197
   - See line  204 
- `cd path/to/barnamekon`
- `heroku local web`
- open this address in your browser localhost:5000 . if `!` appear it means it deployed successfully else, see terminal for log. 
- Go to telegram and interact with Barnamekon.

I will add this to README.md soon.
It's best way I can find. If any problem occurred please write it here, so we will handle that.
Thanks :)

**Comment by HesamKorki** [comment]
Alright thanks. However, I found out that you don't really need flask cause telegram API provides HTTP server for deployment itself. The code would be so much simpler. I am currently working on a fork of the repository. Will do these steps if it was necessary.

**Comment by anvaari** [comment]
Actually, I didn't know anything about that. But If make code simpler, It's definitely better. I will read about that. Do you want to work on replacing Flask with that API ? If no, I will work on it.

**Comment by HesamKorki** [comment]
It would be great if you could make it simpler with that. I tried to host the bot using Ngrok and Heruko local as you mentioned but at the end it was not successful. All I need is a debugging environment so I could test the changes locally.

**Comment by anvaari** [comment]
I know, it is necessary for developing to run bot locally. I work on it and I hope, I can handle this soon.
But until then, can you send me your edited version of Barnamekon.py  and share screen shot of terminal after `heroku local web`?  maybe I can help you with that

**Comment by anvaari** [comment]
I search for it, but can't find smth wich can make bot simpler. We should use webhook to interact with heroku. and [here](https://github.com/eternnoir/pyTelegramBotAPI/tree/master/examples/webhook_examples) are some example which we can use to set up webhook. I think simplest is Flask.
I try [these](https://github.com/anvaari/BarnameKon/issues/4#issuecomment-896857985) steps, and bot run locally. please send me information which I mentioned in last comment to solve problem.
